runtime/shared_common/j9shr.tdf

```diff
@@ -2967,8 +2967,8 @@ TraceEvent=Trc_SHR_CM_updateROMSegmentList_NewHeapAlloc Overhead=1 Level=4 Templ
 
 TraceException=Trc_SHR_CM_storeCacheUniqueID_Failed_To_Get_Manager Overhead=1 Level=1 Template="CM::storeCacheUniqueID(): Failed to get the corresponding manger"
 TraceException=Trc_SHR_CM_storeCacheUniqueID_Failed_To_Store_Prereq_UniqueID Overhead=1 Level=1 Template="CM::storeCacheUniqueID(): Failed to store the prerequisite cache unique ID %.*s"
-TraceEntry=Trc_SHR_CM_storeCacheUniqueID_generateCacheUniqueID_entry Overhead=1 Level=7 Template="CM::storeCacheUniqueID(): Entering generateCacheUniqueID() - (createTime: %llx, metadataBytes: %llx, classesBytes: %llx, lineNumTabBytes: %llx, varTabBytes: %llx) "
-TraceExit=Trc_SHR_CM_storeCacheUniqueID_generateCacheUniqueID_exit Overhead=1 Level=7 Template="CM::storeCacheUniqueID(): Exiting generateCacheUniqueID() - the generated cache unique ID is %.*s"
+TraceEvent=Trc_SHR_CM_storeCacheUniqueID_generateCacheUniqueID_before Overhead=1 Level=7 Template="CM::storeCacheUniqueID(): Entering generateCacheUniqueID() - (createTime: %llx, metadataBytes: %llx, classesBytes: %llx, lineNumTabBytes: %llx, varTabBytes: %llx) "
```

Comment:
If these entry/exit trace points become TraceEvent, please do not use "Entering/Exiting" in the trace message.